corral/bundle/json.pony

```diff
@@ -4,9 +4,10 @@ use "../util"
 
 primitive JsonError
 
-
 primitive Json
-  fun load_object(file_path: FilePath, log: Log): (JsonObject | FileErrNo | JsonError) =>
+  fun load_object(file_path: FilePath, log: Log):
```

Comment:
The ":" at the end of the line here should be on the next line to match the style guide.